tools/gen_struct_info.py

```diff
@@ -231,15 +232,13 @@ def inspect_headers(headers, cpp_opts):
 
   js_file = tempfile.mkstemp('.js')
 
+  shared.check_sanity()
```

Comment:
is the check needed?

Reply:
Its not required but it keeps the output cleaner.    I'll add a comment as to why.

Reply:
Added a comment.